server/channels/api4/apitestlib.go

```diff
@@ -97,6 +97,9 @@ func setupTestHelper(dbStore store.Store, searchEngine *searchengine.Broker, ent
 	*memoryConfig.PluginSettings.ClientDirectory = filepath.Join(tempWorkspace, "webapp")
 	memoryConfig.ServiceSettings.EnableLocalMode = model.NewBool(true)
 	*memoryConfig.ServiceSettings.LocalModeSocketLocation = filepath.Join(tempWorkspace, "mattermost_local.sock")
+	*memoryConfig.LogSettings.EnableSentry = false // disable error reporting during tests
+	*memoryConfig.LogSettings.ConsoleJson = false
+	*memoryConfig.LogSettings.ConsoleLevel = mlog.LvlStdLog.Name
```

Comment:
This will include debug, error, etc...  but I also want to point out it will include "trace".  Looks like we're not using trace much, although I would like to see us use trace when we are just logging that a method was called.  No action - just FYI

Reply:
Even if we add such a verbose trace logging, I wonder if that would be fine to have for unit tests. In the end, every information that helps with debugging tests is useful.